app/assets/stylesheets/base.scss

```diff
@@ -83,8 +83,30 @@ a.active {
   outline: none;
 }
 
+.info-bar {
+  border-radius: 0.75rem;
+  padding: 1.1rem 1.25rem;
+}
+
+.info-bar div .btn {
```

Comment:
Should we try doing something more bootstrappy where we don't scope elements to their parents? Otherwise this button style won't be re-usable elsewhere.